### ISSUE TITLE
Handshapes: add list view listing all handshapes in terms of their features

### ISSUE THREAD

**Issue by ocrasborn** [Signbank/NGT-signbank#346]
* With link on handshape name leading to a detail view for handshapes

**Comment by susanodd** [comment]
![handshapes_list_view](https://user-images.githubusercontent.com/24272544/27958236-64e7726e-6322-11e7-941e-0e0dc6ea3f71.png)

**Comment by susanodd** [comment]
Any preference which of the handshape fields are shown in the list?
I made the image files be names "handshape_" + machine_value of the handshape. There are too many weird characters to have the name be a filename.
I need to work on the query part now.

**Comment by ocrasborn** [comment]
Might it be possible to squeeze in all fields, given the following two suggestions? 
* The image file will be more or less square and need not be as big as the images for glosses
* The columns for Finger selection and Finger configuration (and Fing. sel. 2, Find. conf. 2) could be much narrower, making the field name appear on two lines.

**Comment by susanodd** [comment]
Okay!

**Comment by susanodd** [comment]
I can probably make it so it only shows the name in the language being used.

**Comment by ocrasborn** [comment]
Oh, @susanodd , how are new handshapes added? Do they automatically get added to the handshape table once I add a handshape value in the Django admin under Field values? That would be the desirable behaviour, otherwise the link between signs and handshapes gets lost.

**Comment by susanodd** [comment]
I need to add this.
I had assumed that Handshapes were something like an alphabet, and provision for adding new Handshapes wasn't mentioned in the Issues, so I didn't work on this functionality yet.
I will do it as you suggest.

**Comment by ocrasborn** [comment]
I hadn't thought about it either, but now I see it live, I realise this will be important to implement. (BTW: we will expect quite a few more handshapes in the coming year when we start adding lexicons of other sign languages than NGT.)

**Comment by susanodd** [comment]
Okay, I'm making this be a "lazy" method. So you can add a new handshape via the admin page, under the Field values.
Then, to make the Search Handshapes and the Handshape Detail work for the "new" handshape, this will be created in the Handshapes table on the fly. It will be an empty Handshape just like the other ones, generated from the Fields table, but not until you either query Handshapes or try to view your new handshape.
One issue is, if you use a (new) machine_value for a (new, not yet existing) handshape as an argument to the Handshape Detail url, then this could either be an errror, for a non-existent handshape, or it could lazily create a new handshape for that new machne_value.
(Probably best to let it be an undefined, rather than automatically creating it, I guess. Although this would get around forgetting to add stuff to the Field table, and facilitate implementing scripts etc..)
Or, I guess it could ask whether the user wishes to create a new handshape for the new machine_value.

**Comment by ocrasborn** [comment]
I think we should work from the assumption that a user doesn't know what a machine vale is in the first place, and so will not freely type I machine calls in the url. So just leave it without any consequences (let the system display its standard error page) if s/he does. New values should only be created by those who have access to the admin interface and know how to use it.

**Comment by susanodd** [comment]
Okay!
So once a new Handshape has been created in the Fields table, then the way to "see" it will be via the Handshapes List / Query (it will be created when this list is generated) then when you click on it in the list, you will get to the new Detail View. 
(It will also be in the Handshape choices, so it can end up inside of Gloss data. This still needs to be implemented, that you can navigate to the Handshape data from the Gloss if you want to.)

**Comment by ocrasborn** [comment]
Added a new issue for the latter, #369

**Comment by susanodd** [comment]
This has been fixed. (Pushed to github, master branch.)
